projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -307,6 +310,19 @@ control_service_deploy_testing_data_pipelines:
     - if: '$CI_COMMIT_BRANCH == "main"'
       changes: *control_service_helm_change_locations
 
+control_service_helm_chart_dry_run:
```

Comment:
we have a linting step already. 
What is the difference between dry run and linting? 
Maybe we should remove the linting step if this dry run will catch all those issues?

Reply:
No, both are valuable in my opinion.

* `helm lint` inspects the **entire** chart, checking every file and template for potential style and syntax issues. 
* With `helm install --dry-run` we are rendering the templates for the particular configuration provided by our values.yaml/--set options only.  It will catch runtime issues and issues with rendering the templates which lint cannot. But only on the configured path.